lib/registries/addon/components/registries-license-picker/template.hbs

```diff
@@ -35,25 +35,24 @@
         </div>
         
         {{#if @manager.selectedLicense}}
-            {{#if @manager.requiredFields}}
-                <br>
-            {{/if}}
             <form.custom
+                local-class='AdditionalFields'
                 @changeset={{@manager.changeset}}
                 @valuePath='nodeLicense'
             >
                 {{#each @manager.requiredFields as |key|}}
                     <div class='form-group'>
-                        <label>
+                        <label for={{key}}>
```

Comment:
this should match the `id` of the input, which has `license-` prepended.

also, tho, the `unique-id` helper is meant for this sort of thing -- thoughts on using that instead of building the `id`?

Reply:
Talked to Futa in person a bit about this and decided that we can probably move the input back into the label for now to fix this problem and to keep the textfields a consistent size with how they normally are, since moving them out of the label makes them the size of a normal text field.

Reply:
With the Year and Copyright holders text input being half-size vs full-size, that may be a question for product, so we should ask them at some point. A better way to indicate that those two fields are part of the license (both visually and semantically) would be nice, but for now I think moving them back in and making them (Year + Copyright Holder) half-width again is good